docs/source/markdown/options/no-hostname.md

```diff
@@ -1,9 +1,9 @@
 ####> This option file is used in:
-####>   podman build, farm build
+####>   podman build, create, farm build, kube play, pod create, run
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
 #### **--no-hostname**
 
-Do not create the _/etc/hostname_ file in the container for RUN instructions.
+Do not create the _/etc/hostname_ file in the containers.
 
-By default, Buildah manages the _/etc/hostname_ file, adding the container's own hostname.  When the **--no-hostname** option is set, the image's _/etc/hostname_ will be preserved unmodified if it exists.
+By default, Podman manages the _/etc/hostname_ file, adding the container's own hostname.  When the **--no-hostname** option is set, the image's _/etc/hostname_ will be preserved unmodified if it exists.
```

Comment:
This may be a bit confusing, while the description clearly states that it only effects the file it may be worth highlighting that setting a hostname is still supported as the gethostname() syscall can still be used in the container to get the real hostname.